nservicebus/operations/performance-counters-criticaltime.include.md

```diff
@@ -4,12 +4,12 @@
 
 **Added in:** Version 3
 
-Critical Time is the time from a message being sent until successfully processed. This metric is useful for monitoring message Service-level agreement. For example "All orders should be processed within X seconds/minutes/hours". Define an SLA for each endpoint and use the `CriticalTime` counter to ensure it is adhered to.
+Critical Time is the time from a message being sent until successfully processed. This metric is useful for monitoring message Service-level agreement. For example "All orders should be processed within X seconds/minutes/hours". [Define an SLA](performance-counters.md#sla-violation-countdown) for each endpoint and use the `CriticalTime` counter to ensure it is adhered to.
```

Comment:
The link here seems to fail the content verification. I don't think it is necessary. Remove?